e3sm_diags/parameter/core_parameter.py

```diff
@@ -96,7 +98,9 @@ def __init__(self):
         # Diagnostic plot settings
         # ------------------------
         self.main_title: str = ""
-        self.backend: str = "mpl"
+        # TODO: Remove `backend` because it is always e3sm_diags/plot/cartopy.
+        # This change cascades down to changes in `e3sm_diags.plot.plot`.
+        self.backend: str = "cartopy"
```

Comment:
Good call!

Reply:
Yeah, the plan is to move all of the plots out of `plots/cartopy` one level down into `plots`.

I started that transition with `lat_lon_plot.py`, `lat_lon_land_plot.py` and `lat_lon_river_plot.py`.